platform/security/servlet/security-servlet-logout/src/main/java/org/codice/ddf/security/servlet/logout/LogoutService.java

```diff
@@ -80,17 +80,14 @@ public Response getActionProviders(@Context HttpServletRequest request)
                             "",
                             true);
 
-                    if (displayName != null
```

Comment:
The second parameter of `SubjectUtils.getName` provides a default value.  Display name should never be null at this point.